### PR TITLE
Playwright_PEPPER_1300_Check OS1 to OS2 participant display

### PR DESCRIPTION
**Pull request by Rinchan20**
For this ticket: https://broadworkbench.atlassian.net/browse/PEPPER-1300 which check the display of re-consented osteo participants

### CODE REVIEW

File: playwright-e2e/dsm/component/customize-view-column.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Locator, Page, expect } from '@playwright/test';
+import { CustomizeView, CustomizeViewID, Label } from 'dsm/enums';
+
+export default class ColumnGroup {
```
Comment (Rinchan20) [inline playwright-e2e/dsm/component/customize-view-column.ts:4]: implementing feedback from other PR to make this a component

**Comment by Rinchan20** [comment]
looks a bit long - more tests to add - currently takes about 27 seconds -> 30 seconds to finish running

**Comment by Rinchan20** [comment]
ready for review but going to run all tests in feature branch first just to make sure something wasn't forgotten

File: playwright-e2e/tests/dsm/osteo/os1-to-os2-display-verification.spec.ts [hunk 2]
```diff
@@ -0,0 +1,822 @@
...
+      participantListTable = participantListPage.participantListTable;
+      const participantPage = await participantListTable.openParticipantPageAt(0);
+
+      oncHistoryTab = await participantPage.tablist(Tab.ONC_HISTORY).click<OncHistoryTab>();
+      const oncHistoryTable: OncHistoryTable = oncHistoryTab.table;
+
+      //There is usually an extra row that is empty and ready for input
+      const numberOfRows = await oncHistoryTable.getRowsCount() - 1;
+      expect(numberOfRows).toBeGreaterThanOrEqual(1);
+      /**
```
Comment (zyme) [inline playwright-e2e/tests/dsm/osteo/os1-to-os2-display-verification.spec.ts:420]: The code below is so clear that I don't think we need this comment.

**Review by zyme — APPROVED** [review]
Good stuff!

**Comment by Rinchan20** [comment]
running all tests to make sure no other tests broke - and then merging in

**Pull request merged**